site/content/en/blog/releases/1.22.0-rc

```diff
@@ -0,0 +1,49 @@
+---
+title: "1.22.0 - Release Candidate"
+linkTitle: "1.22.0-rc"
+date: "2022-03-22"
+---
+This is the 1.22.0-rc release of Agones.
+
+This Release  fixes the issue with  agones-images on ARM systems, multi-cluster allocation example solution to leverage GCP resources to connect to multiple Agones clusters and also Cert manager integration
+
+Check the <a href="https://github.com/googleforgames/agones/tree/release-1.22.0-rc" data-proofer-ignore>README</a> for details on features, installation and usage.
+
+**Implemented enhancements:**
+
+- Add a multi-cluster allocation example solution to leverage GCP resources to connect to multiple Agones clusters [\#2495](https://github.com/googleforgames/agones/issues/2495)
+- Support Agones on ARM systems [\#2216] (https://github.com/googleforgames/agones/issues/2216)
```

Comment:
I reopened this issue, since it was accidentally closed. This work is not yet complete, it's only been started.

We should remove this line from the release notes and Changelog (if the Changelog was run before I reopened the issue).

Reply:
I just noticed this was resolved, but it looks like the ARM issue (#2216) is still listed here and on the CHANGELOG as completed, when it's not. Maybe you removed a different one? (Apologies if I wasn't clear).